hail/www/references.md

```diff
@@ -0,0 +1,90 @@
+
+# Scientific Output from the Hail Community  
```

Comment:
I'm not sure we want to presume that everyone who uses Hail in their work considers themselves part of the Hail community. What about a title like "Hail-powered Science"?